roles/ssp/defaults/main.yml

```diff
@@ -2,11 +2,14 @@
 #
 ---
 
-ssp_version: "1.17"
+ssp_version: "2.1.0"
 
-ssp_repo_url: "git://github.com/simplesamlphp/simplesamlphp.git"
+ssp_repo_url: "https://github.com/simplesamlphp/simplesamlphp.git"
 ssp_repo_version: "simplesamlphp-{{ ssp_version }}"
-ssp_path: "/var/simplesamlphp"
+# Uncomment in order to install using the release
+# ssp_release_url: "https://github.com/simplesamlphp/simplesamlphp/releases/download/v{{ ssp_version }}/{{ ssp_repo_version }}-full.tar.gz"
+ssp_basepath: "/var/simplesamlphp"
```

Comment:
The `ssp_basepath` can be omitted. Instead it can be created with the tempfile module in `install-release.yml` (see suggested version below)

Reply:
done